src/Donations/PaymentMethods/SepaPayments.tsx

```diff
@@ -66,7 +73,10 @@ function SepaPayments({
     });
   };
 
-  const createPaymentMethodSepa = (sepaElement: any, contactDetails: any) => {
+  const createPaymentMethodSepa = (
+    sepaElement: any,
```

Comment:
@sunilsabatp should the type be `any` here?

Reply:
@prachigarg19 is it a contactDetails ? type

Reply:
@prachigarg19  is it the contactDetails ?